### DISCUSSION TITLE
[A/L] (2021/22) Funzioni - Esercizio 18

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#258] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152315967-7d8f35b6-b0a0-47ea-a448-a8bdc993a024.png)

**Comment by Rurik-D** [comment]
.V
.V
.V

Le prime due seguono perfettamente la definizione di intersezione e unione su funzioni.
La terza è vera in quanto è vero che l'intersezione delle pre-immagini è sottoinsieme dell'unione delle immagini, ma non è vero il contrario. 
Dire quindi che f(A) u f(B) ⊆ f(AnB) sarebbe stato sbagliato.

**Comment by LuginiAndrea** [comment]
> .V .V .V
> 
> Le prime due seguono perfettamente la definizione di intersezione e unione su funzioni. La terza è vera in quanto è vero che l'intersezione delle pre-immagini è sottoinsieme dell'unione delle immagini, ma non è vero il contrario. Dire quindi che f(A) u f(B) ⊆ f(AnB) sarebbe stato sbagliato.

La seconda è falsa.
Basta infatti considerare $f\left(x\right) = x^2, f: \mathbf{Z} \rightarrow \mathbf{N}$ e prendere come insiemi 
$A = -2, 4$ e $B = -4, 2$ la cui intersezione è $A \cap B = \emptyset$, ma l'intersezione delle immagini è $f\left(A\right) \cap f\left(B\right) = 4, 16 \cap 4, 16 = 4, 16 \neq f\left(\emptyset\right)$

**Comment by LuginiAndrea** [comment]
La prima è vera:
Consideriamo che, nel mondo insiemistico, $X = Y \Longleftrightarrow X \subset Y, Y \subset X$
Dimostriamo $f\left(A \cup B\right) \subset f\left(A\right) \cup f\left(B\right)$
$\forall x \in A \cup B$ sappiamo che, per ovvi motivi, $x \in A \vee x \in B$. Unito al fatto che 
$$\forall y \in f\left(A \cup B\right) \exists x \in A \cup B \textrm{ t.c. } f\left(x\right) = y$$
Ne consegue che $\forall y \in f\left(A \cup B\right) \exists x \in A \vee \exists x \in B \textrm{ t.c. } f\left(x\right) = y \Longrightarrow y \in f\left(A\right) \vee y \in f\left(B\right) \Longrightarrow \forall y \in f\left(A \cup B\right) y \in f\left(A\right) \cup f\left(B\right)$$
Dimostriamo l'inclusione nell'altro senso
$$\forall y \in f\left(A\right) \cup f\left(B\right) \exists x \in A \vee \exists x \in B \textrm{ t.c. } f\left(x\right) = y \Longrightarrow \exists x \in A \cup B \textrm{ t.c. } f\left(x\right) = y \Longrightarrow y \in f\left(A \cup B\right) \Longrightarrow \forall y \in f\left(A\right) \cup f\left(B\right) y \in f\left(A \cup B\right)$$

La seconda è falsa.
Basta infatti considerare $f\left(x\right) = x^2, f: \mathbf{Z} \rightarrow \mathbf{N}$ e prendere come insiemi 
$A = -2, 4$ e $B = -4, 2$ la cui intersezione è $A \cap B = \emptyset$, ma l'intersezione delle immagini è $f\left(A\right) \cap f\left(B\right) = 4, 16 \cap 4, 16 = 4, 16 \neq f\left(\emptyset\right)$

La terza è vera.
Infatti, $\forall x \in A \cap B$ sappiamo che $x \in A \wedge x \in B$. Ne consegue che $x \in A$ è sempre vera.
Quindi $$\forall y \in f\left(A \cap B\right) \exists x \in A \wedge \exists x \in B \textrm{ t.c. } f\left(x\right) = y \Longrightarrow \forall y \in f\left(A \cap B\right) \exists x \in A \textrm{ t.c. } f\left(x\right) = y \Longrightarrow y \in f\left(A\right)$$
Visto che l'unione è un operazione additiva ne consegue che il sol fatto che $y \in f\left(A\right)$ implica che $y \in f\left(A\right) \cup f\left(B\right)$
